profiler/src/ProfilerEngine/DeployResources/DDProf-SetEnv.sh

```diff
@@ -14,11 +14,11 @@ fi
 
 export CORECLR_ENABLE_PROFILING=1
 export CORECLR_PROFILER={BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}
-export CORECLR_PROFILER_PATH_64=${DD_DEPLOY_DIRECTORY}/Datadog.Profiler.Native.so
+export CORECLR_PROFILER_PATH_64=${DD_DEPLOY_DIRECTORY}/linux-x64/Datadog.Profiler.Native.so
```

Comment:
I assume this script isn't used on alpine or arm64 then? 🤔

Reply:
Exactly. I was thinking about it but this script is only used on linux-x64 for our sanitizer jobs

Reply:
When having those jobs on arm64 or alpine I will update the script.